common/shlibs

```diff
@@ -4185,3 +4185,41 @@ libnvidia-container-go.so.1 libnvidia-container-1.10.0_1
 libqtspell-qt5.so.1 qtspell-1.0.1_1
 libopensmtpd.so.0 libopensmtpd-0.7_1
 libiio.so.0 libiio-0.23_1
+libcivetweb.so.1 libcivetweb-1.15_1
+libcmr.so.17
+libcmr.so.17 libdcmtk-3.6.7_1
+libdcmrt.so.17 libdcmtk-3.6.7_1
+libdcmdata.so.17 libdcmtk-3.6.7_1
+libdcmseg.so.17 libdcmtk-3.6.7_1
+libdcmdsig.so.17 libdcmtk-3.6.7_1
+libdcmsr.so.17 libdcmtk-3.6.7_1
+libdcmect.so.17 libdcmtk-3.6.7_1
+libdcmtkcharls.so.17 libdcmtk-3.6.7_1
+libdcmfg.so.17 libdcmtk-3.6.7_1
+libdcmtls.so.17 libdcmtk-3.6.7_1
+libdcmimage.so.17 libdcmtk-3.6.7_1
+libdcmtract.so.17 libdcmtk-3.6.7_1
+libdcmimgle.so.17 libdcmtk-3.6.7_1
+libdcmwlm.so.17 libdcmtk-3.6.7_1
+libdcmiod.so.17 libdcmtk-3.6.7_1
+libi2d.so.17 libdcmtk-3.6.7_1
+libdcmjpeg.so.17 libdcmtk-3.6.7_1
+libijg12.so.17 libdcmtk-3.6.7_1
+libdcmjpls.so.17 libdcmtk-3.6.7_1
+libijg16.so.17 libdcmtk-3.6.7_1
+libdcmnet.so.17 libdcmtk-3.6.7_1
+libijg8.so.17 libdcmtk-3.6.7_1
+libdcmpmap.so.17 libdcmtk-3.6.7_1
+liboflog.so.17 libdcmtk-3.6.7_1
+libdcmpstat.so.17 libdcmtk-3.6.7_1
+libofstd.so.17 libdcmtk-3.6.7_1
+libdcmqrdb.so.17 libdcmtk-3.6.7_1
+libDelayedDeletion.so.1.11.1 Orthanc-plugins-1.11.1_1
```

Comment:
common/shlibs is meant to only contain solibs used by otehr packages - are plugins like that?

Reply:
these plugins will probably never be used by other pkgs, I removed its from shlibs